docker/witnet-rust/Dockerfile

```diff
@@ -3,7 +3,8 @@ FROM --platform=$TARGETPLATFORM ubuntu:focal
 # Install basic environment dependencies
 RUN apt-get update && apt-get install -y --no-install-recommends \
     ca-certificates \
-    curl
+    curl \
+    netcat
```

Comment:
I don't see `netcat` being used in any script

Reply:
Oh, I found it, it's used as `nc` to check if the node JSON-RPC is working.

Reply:
It's called `nc`